code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -19,6 +19,7 @@
 	response_help  = "pets"
 	response_disarm = "bops"
 	response_harm   = "kicks"
+	mob_size = MOB_SIZE_HUMAN

```

Comment:
Human was just the next size up I could think of off the top of my head. they need to be bigger than small or they can't open doors.